<launch>
  <node pkg="tf_republisher" type="tagslam_tf_republisher.py"
    name="fisheye_tf_broadcaster" output="screen">
    <param name="tagslam_base_frame" type="string" value="tagslam_base"/>
    <param name="tagslam_map_frame" type="string" value="map"/>
  </node>

  <node name="rqt_reconfigure_fisheye" pkg="rqt_reconfigure" type="rqt_reconfigure"/>
  <node pkg="tf_republisher" type="calibrate_tf.py"
    name="fisheye_tf_calibrator" output="screen">
    <param name="source_frame" type="string" value="Fisheye_Model"/>
    <param name="map_frame" type="string" value="map"/>
    <param name="save_file" type="string"
      value="$(find tf_republisher)/config/fisheye_position.yaml"/>
  </node>
</launch>
